app/assets/javascripts/modules/cross-domain-tracking.js

```diff
@@ -0,0 +1,31 @@
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  'use strict'
+  Modules.CrossDomainTracking = function () {
+    this.start = function ($context) {
+      var trackableLinkSelector = '[href][data-tracking-code][data-tracking-name]';
+      var $element;
```

Comment:
Any reason $context isn't $element? It looks like `$element` isn't being used in this scope.

Reply:
Like https://github.com/alphagov/static/blob/master/spec/javascripts/modules/track-click.spec.js#L33-L40

I think this makes it clearer in the tests how to use the module.

Reply:
Looks like I left this in, I should configure standard.js in my editor!